config/site_settings.yml

```diff
@@ -744,6 +744,14 @@ posting:
     default: 0
     locale_default:
       fr: 1
+  enable_inline_emoji_translation:
+    client: true
+    default: true
```

Comment:
It's the opposite. It should be `false` by default and enabled only for CJK locales.